hash/zx81_cass.xml

```diff
@@ -2465,7 +2570,7 @@ RAND USR 16530
 			</dataarea>
 		</part>
 	</software>
-
+	<!--
 	<software name="flsimultx" cloneof="flsimul">
 		<description>The Flight Simulator (Timex)</description>
 		<year>1982</year>
```

Comment:
Is there a good reason to keep these commented rather than just removing them?  You could add “Flug-Simulation” as an `alt_title` for the parent, and add a comment to say it was also distributed by Timex (presumably for the US market).

Reply:
No particular reason to keep them. So I removed the 2 commented sets.